src/spaces/continuous.jl

```diff
@@ -11,32 +11,39 @@ Base.eltype(s::ContinuousSpace{D,P,T,F}) where {D,P,T,F} = T
 defvel(a, m) = nothing
 
 """
-    ContinuousSpace(extent::NTuple{D, <:Real}, spacing = min(extent...)/10; kwargs...)
+    ContinuousSpace(extent::NTuple{D, <:Real}; kwargs...)
 Create a `D`-dimensional `ContinuousSpace` in range 0 to (but not including) `extent`.
-`spacing` configures the compartment spacing that the space is divided in, in order to
-accelerate nearest neighbor functions like [`nearby_ids`](@ref).
-All dimensions in `extent` must be completely divisible by `spacing` (i.e. no
-fractional remainder).
 Your agent positions (field `pos`) must be of type `NTuple{D, <:Real}`,
-use [`ContinuousAgent`](@ref) for convenience.
-In addition it is useful for agents to have a field `vel::NTuple{D, <:Real}` to use
-in conjunction with [`move_agent!`](@ref).
-
-The keyword `periodic = true` configures whether the space is periodic or not. If set to
-`false` an error will occur if an agent's position exceeds the boundary.
-
-The keyword argument `update_vel!` is a **function**, `update_vel!(agent, model)` that updates
-the agent's velocity **before** the agent has been moved, see [`move_agent!`](@ref).
-You can of course change the agents' velocities
-during the agent interaction, the `update_vel!` functionality targets spatial force
-fields acting on the agents individually (e.g. some magnetic field).
-By default no update is done this way.
-If you use `update_vel!`, the agent type must have a field `vel::NTuple{D, <:Real}`.
-
-There is no "best" choice for the value of `spacing`. If you need optimal performance it's
-advised to set up a benchmark over a range of choices. The value matters most when searching
-for neighbors. In [`Models.flocking`](@ref) for example, an optimal value for `spacing` is
-66% of the search distance.
+while it is strongly recommend that agents also have a field `vel::NTuple{D, <:Real}` to use
+in conjunction with [`move_agent!`](@ref). Use [`ContinuousAgent`](@ref) for convenience.
+
+`ContinuousSpace` is a _true_ representation of agent dynamics on a continuous medium.
+There are no "patches". Agent position, orientation, and speed, are true floats.
```

Comment:
> There are no "patches".

I'd probably drop this from our docstring. I plan to write some documentation around 'Moving from framework X to Agents.jl' at some stage, this kind of thing should be there. Patches have no context outside of Netlogo regardless.